src/test/kotlin/uk/gov/dluhc/emsintegrationapi/database/repository/ApprovedPostalVoteApplicationRepositoryTest.kt

```diff
@@ -0,0 +1,25 @@
+package uk.gov.dluhc.emsintegrationapi.database.repository
+
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Test
+import org.springframework.beans.factory.annotation.Autowired
+import uk.gov.dluhc.emsintegrationapi.testsupport.testdata.buildApprovedPostalApplication
+
+class ApprovedPostalVoteApplicationRepositoryTest : AbstractRepositoryTest() {
```

Comment:
should the name of this class and its parent have the word "Integration" at least to tell us it's a integration test?

Reply:
Done